dpc-sdp/ripple
# [SDPA-5940] Made search scroll to top when pagination changes

## Motivation and Context



**JIRA issue:** https://digital-engagement.atlassian.net/browse/SDPA-5940

## Changed



1.Added ability for search page to scroll to top of search element when pagination changes

### Screenshots



## How Has This Been Tested?





Using `reference:dev`
1. Navigate to home page
2. Using the search, look up "demo"
3. Scroll to bottom of search results and change page

The page should scroll to the top of results

## Types of changes



- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist




- [ ] I've added relevant changes to the documentation.
- [ ] I have added tests to cover my changes (if not applicable, please state why)
- [ ] My change requires a template update for create-ripple-app.
- [ ] I have added template update script for next release.
